scripts/pull.sh

```diff
@@ -77,6 +78,14 @@ while (( "$#" )); do
         NO_AUTH=true
         shift
       ;;
+    -vv|--verbose)
+        VERBOSITY="-vv"
+        shift
+      ;;
+    -vvv)
```

Comment:
Nice @ethangeorgi, minor comment: might want to add`--debug` as well to this switch statement, I think that is the new Lando option for max verbosity (old ones still work, but `--debug` will probably be emphasized more going forward).

Reply:
Okay, I didn't know about `--debug` (and can't find it in the docs, or in `terminus --help`). I'm hoping it's the same thing as `-vvv`. Easy enough to add!